client/blocks/login/two-factor-authentication/waiting-notification-approval.jsx

```diff
@@ -80,42 +79,44 @@ class WaitingTwoFactorNotificationApproval extends Component {
 		} = this.props;
 
 		return (
-			<form>
-				<Card className="two-factor-authentication__push-notification-screen is-compact">
-					<p>
-						{ translate( 'We just sent a push notification to your WordPress mobile app. ' +
-							'Swipe or tap to open and verify your login.' ) }
-					</p>
-					<div>
-						<img className="two-factor-authentication__auth-code-preview"
-							src="/calypso/images/login/pushauth.svg" />
-					</div>
-				</Card>
-				<Card className="two-factor-authentication__form-action is-compact">
-					<p>
-						{ translate( 'Or, type in the verification code generated by your Authenticator app' +
-							' or sent via text.' ) }
-					</p>
-					<p>
-						<Button onClick={ this.verifyWithCodeInstead }>
-							{ translate( 'Verify with code instead' ) }
-						</Button>
-					</p>
-				</Card>
-				<p>
-					<ExternalLink
-						icon={ true }
-						target="_blank"
-						href="http://en.support.wordpress.com/security/two-step-authentication/">
-						{ translate( 'Help' ) }
-					</ExternalLink>
-				</p>
-				{ isSmsAuthSupported && (
-					<p>
-						<a href="#" onClick={ this.sendSmsCode }>{ translate( 'A recovery code via text' ) }</a>
-					</p>
-				) }
-			</form>
+			<div>
+				<div className="two-factor-authentication__header">
+					{ this.props.title }
+				</div>
+
+				<form>
+					<Card className="two-factor-authentication__push-notification-screen is-compact">
+						<p>
+							{ translate( 'We sent a push notification to your {{strong}}WordPress mobile app{{/strong}}. ' +
+								'Once you get it and swipe or tap to confirm, this page will update.', {
+									components: {
+										strong: <strong />
+									}
+								} ) }
+						</p>
+						<div>
+							<img className="two-factor-authentication__auth-code-preview"
+								src="/calypso/images/login/pushauth.svg" />
+						</div>
+					</Card>
+					<Card className="two-factor-authentication__form-action is-compact">
+						<p>
+							{ translate( 'Or continue to your account using:' ) }
+						</p>
+						<p>
+							<a href="#" onClick={ this.verifyWithCodeInstead }>
+								{ translate( 'An Authenticator App' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **4** times:
`translate( 'An Authenticator application' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).